src/System Application/App/Azure AD Plan/src/Plan Configuration/PlanConfiguration.Table.al

```diff
@@ -54,4 +54,24 @@ table 9017 "Plan Configuration"
             Unique = true;
         }
     }
+    trigger OnDelete()
```

Comment:
I don't think this is sufficient. Aren't these only run if fx `PlanConfiguration.Modify(true)` is run?

It'd be safe to user the database events. OnBefore/AfterDelete events etc

Reply:
Will take it in the other PR.